README.md

```diff
@@ -94,12 +94,12 @@ We also provide a [shaded JAR](manual/shaded_jar/)
 to avoid the explicit dependency to Netty.
 
 If you can't use a dependency management tool, a
-[binary tarball](http://downloads.datastax.com/java-driver/cassandra-java-driver-3.11.4.tar.gz)
+[binary tarball](http://downloads.datastax.com/java-driver/cassandra-java-driver-3.11.5.tar.gz)
```

Comment:
Note the 3.11.4 version of this link is not found

Reply:
That is... interesting.

I wonder if we need to do something separate there to fix that.  I don't remember anything in the build process calling out for that... this may need some follow-up investigation.